classes/class-kss-shipping-method.php

```diff
@@ -41,11 +41,21 @@ public function __construct( $instance_id = 0 ) {
 		 */
 		public function init_form_fields() {
 			$this->instance_form_fields = array(
-				'title' => array(
+				'title'      => array(
 					'title'       => __( 'Klarna Shipping Assistant', 'klarna-shipping-service-for-woocommerce' ),
 					'type'        => 'title',
 					'description' => __( 'There are currently no settings for Klarna Shipping Assistant since this is controlled by the TMS-provider. If other plugins adds settings, these are shown below.', 'klarna-shipping-service-for-woocommerce' ),
```

Comment:
Perhaps we should tweak the description slightly since the shipping method now have a setting?